### DISCUSSION TITLE
1.24.0 beta

### DISCUSSION THREAD

**Discussion by jamie-mh** [stratumauth/app#1053] [General]
- **New feature**: Skip to next code
- **New feature**: Use system font
- **General**: Code is copied only when code is revealed
- **General**: 58 new icons and improvements to existing icons
- **General**: Translation updates
- **General**: Bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/jamie-mh/AuthenticatorPro/releases/tag/1.24.0-beta'>1.24.0 beta</a>.</em>

**Comment by OmarNazzal150** [comment]
Hi. when 1.24.0 gets released on the Play store. Also, check out these old and newer issues


#288

#369

#514

#593

#770

#838

#916

#966

#1024

#1047
